UI/RetroAchievementScreens.cpp

```diff
@@ -242,13 +241,18 @@ void RetroAchievementsSettingsScreen::CreateAccountTab(UI::ViewGroup *viewGroup)
 		} else {
 			// Hack up a temporary quick login-form-ish-thing
 			viewGroup->Add(new PopupTextInputChoice(&username_, di->T("Username"), "", 128, screenManager()));
-			viewGroup->Add(new PopupTextInputChoice(&password_, di->T("Password"), "", 128, screenManager()));
-			viewGroup->Add(new Choice(di->T("Log in")))->OnClick.Add([=](UI::EventParams &) -> UI::EventReturn {
+			viewGroup->Add(new PopupTextInputChoice(&password_, di->T("Password"), "", 128, screenManager()))->SetPasswordDisplay();
+			Choice *loginButton = viewGroup->Add(new Choice(di->T("Log in")));
+			loginButton->OnClick.Add([=](UI::EventParams &) -> UI::EventReturn {
 				if (!username_.empty() && !password_.empty()) {
 					Achievements::LoginAsync(username_.c_str(), password_.c_str());
+					password_.clear();
```

Comment:
I suppose, ideally, should memset() to zero first before clear().  Though, we're not even using SSL yet so probably that's overthinking things.

-[Unknown]

Reply:
Though, why not. I'll do it.